Refrigerator.cs

```diff
@@ -0,0 +1,315 @@
+﻿using Refrigerator_EX;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Refrigerator_EX
+{
+    class Refrigerator
+    {
+        private static int nextID = 1;
+
+        private int refrigeratorID;
+        private string model;
+        private string color;
+        private int numberOfShelves;
+        private List<Shelf> shelves;
+
+        public Refrigerator( string model, string color, int numberOfShelves)
+        {
+            this.refrigeratorID = nextID++;
+            this.model = model;
+            this.color = color;
+            this.numberOfShelves = numberOfShelves;
+            this.shelves = new List<Shelf>();
+        }
+
+        public Refrigerator()
+        {
+            this.refrigeratorID = nextID++;
+        }
+
+        public int RefrigeratorID
+        {
+            get { return refrigeratorID; }
+            set {; } 
+        }
+        public string Model 
+        {
+            get { return model; }
+            set {
+                if (!(value.Equals("")))
+                    model = value;
+                else
+                    Console.WriteLine("model must be contain string");
+            }
+        }
+        public string Color
+        {
+            get { return color; }
+            set
+            {
+                if (!(value.Equals("")))
+                    color = value;
+                else
+                    Console.WriteLine("color must be contain string");
+            }
+        }
+        public int NumberOfShelves 
+        {
+            get { return numberOfShelves; }
+            set { if (value > 0 && value < 15)
+                    numberOfShelves = value;
+                else
+                    Console.WriteLine("The number of shelves must be positive less than 15");
+                    }
+        }
+        public List<Shelf> Shelves
+        {
+            get { return shelves; }
```

Comment:
why are you exposing the original list? it can be modified by whoever gets it destroying encapsulation.